src/controllers/identity-controllers/participant-controller.ts

```diff
@@ -0,0 +1,85 @@
+import { EdcConnectorClientContext } from "../../context";
+import {
+  ParticipantInput,
+  ParticipantRoleResponse,
+} from "../../entities/participant";
+import { Inner } from "../../inner";
+
+export class ParticipantController {
+  #inner: Inner;
+  #context?: EdcConnectorClientContext;
+
+  constructor(
+    inner: Inner,
+    public participantId: string,
+    context?: EdcConnectorClientContext,
+  ) {
+    this.#inner = inner;
+    this.#context = context;
+  }
+
+  async delete(context?: EdcConnectorClientContext) {
+    const actualContext = context || this.#context!;
+
+    return this.#inner.request<string>(actualContext.identity, {
+      path: `/v1alpha/participants/${this.participantId}`,
```

Comment:
nit: the path first part could be kept in a class field, that will make it easier to spot and to update when it'll be needed (when it'll reach the `beta` and then `stable` phase)

Reply:
will update this on the other prs as well